source/key-concepts/input_data.md

```diff
@@ -1,8 +1,9 @@
 # Input Data
 
-  You can use OpenFisca with two kind of input information:
-  - either *test case*: you simulate the legislation for one standard situation
-  - or *data*: you give a whole population (survey with aggregated data or csv files for example) on which you want to apply the legislation.
+You can use OpenFisca with two kind of input information:
+
+- either *test cases*: you simulate the legislation for one standard situation
+- or *data*: you give a whole population (survey with aggregated data, csv files, etc.) on which you want to apply the legislation
```

Comment:
you give -> you provide

Reply:
What about `bulk data`

Reply:
@guillett replacing what? `data` is already plural for `datum`, and I think `population` is clearer than `bulk`.

Reply:
I propose :

> - or *data*: you provide a population (survey with aggregated data, CSV files with bulk data, etc.) on which you want to apply the legislation

Reply:
Well, for me the two input types are
- Structured data, a JSON payload and
- Tabular data, CSV files

Reply:
If you mean replacing both *test cases*  and *data* with *structured data, a JSON payload* and
*tabular data, CSV files*, later in the doc you have:

> Test cases can be expressed in Python, or in JSON when using the Web API

Reply:
As a current informed user, the two alternatives do not describe properly the two possible usages.

> You can use OpenFisca with two sort of input format structured or tabular data.
> - Structured payload are a good way to describe and simulate the legislation to simple scenarios;
> - Tabular data are particularly relevant for big datasets (from third party systems for instance)

That's a proposal.

Reply:
Thanks, it is clearer now what you meant.